csrankings-s.csv

```diff
@@ -1834,7 +1834,6 @@ Stefan Tai,TU Berlin,https://www.ise.tu-berlin.de/menue/team/prof_dr_ing_stefan_
 Stefan Vandewalle,KU Leuven,https://people.cs.kuleuven.be/~stefan.vandewalle,Y-en1DsAAAAJ
 Stefan Voss,University of Hamburg,https://www.bwl.uni-hamburg.de/iwi/team/mitglieder/stefan-voss.html,PuMO0YsAAAAJ
 Stefan Voß,University of Hamburg,https://www.bwl.uni-hamburg.de/iwi/team/mitglieder/stefan-voss.html,PuMO0YsAAAAJ
-Stefan Wagner 0001,University of Stuttgart,http://www.iste.uni-stuttgart.de,tXL31ZEAAAAJ

```

Comment:
Please update with new affiliation (TUM); please do the same for any applicable faculty who have moved to CS departments in CSrankings-indexed institutions.

Reply:
Thanks for the comment. I have updated the new affiliations.